tools/backups2datalad/__main__.py

```diff
@@ -116,18 +116,34 @@ async def main(
         " backup, delete them instead of erroring"
     ),
 )
+@click.option(
+    "--mode",
+    type=click.Choice(list(Mode)),
+    default=None,
+    help=(
+        "How to decide whether to back up a Dandiset.  'timestamp' — only if"
+        " timestamp of last backup is older than modified timestamp on server;"
+        " 'force' — always; 'verify' — always, but error if there are any"
+        " changes without a change to the timestamp"
```

Comment:
so what is the default if `default=None`?  similarly for `--zarr-mode`.  --help is not helpful on that:

```
  --mode [timestamp|verify|force]
                                  How to decide whether to back up a Dandiset.
                                  'timestamp' — only if timestamp of last
                                  backup is older than modified timestamp on
                                  server; 'force' — always; 'verify' — always,
                                  but error if there are any changes without a
                                  change to the timestamp
  --tags / --no-tags              Enable/disable creation of tags for releases
                                  [default: enabled]
  -w, --workers INTEGER           Number of workers to run in parallel
  --zarr-mode [timestamp|checksum|force]
                                  How to decide whether to back up a Zarr.
                                  'timestamp' — only if timestamp of last
                                  backup is older than some Zarr entry in
                                  S3;'checksum' — only if Zarr checksum is out
                                  of date or doesn't match expected value;
                                  'force' — always
```

Reply:
The true defaults are set in `config.py` (they're both `timestamp`), but (like all config options) they can be set in the config file, and then an explicit option given on the command line overrides the config file, but the latter is only implementable if the CLI default is `None`.

I'll make the defaults explicit in the `--help`.

Reply:
Thank you!